tensorflow_lite_support/python/task/core/proto/BUILD

```diff
@@ -12,3 +12,15 @@ py_library(
     srcs = ["configuration_pb2.py"],
     deps = ["@org_tensorflow//tensorflow/lite/experimental/acceleration/configuration:configuration_proto_external_py"],
 )
+
+py_library(
+    name = "class_pb2",
+    srcs = ["class_pb2.py"],
+    deps = ["//tensorflow_lite_support/cc/task/vision/proto:class_py_pb2"],
+)
+
+py_library(
+    name = "classifications_pb2",
+    srcs = ["classifications_pb2.py"],
+    deps = ["//tensorflow_lite_support/cc/task/vision/proto:classifications_py_pb2"],
+)
```

Comment:
Sorry, the final decision we've reached is that those should live under python/task/processor/proto.

Reply:
alright, will do

Reply:
@mbrenon Fixed it!